rhizome/host/bin/prep_host.rb

```diff
@@ -4,6 +4,13 @@
 require_relative "../../common/lib/util"
 require_relative "../lib/cloud_hypervisor"
 require "fileutils"
+require "socket"
+
+original_hostname = Socket.gethostname
+hostname = ARGV.shift
```

Comment:
We usually raise exception if an expected parameter is not passed. Either like rhizome/host/bin/prepvm.rb or like rhizome/common/bin/daemonizer